cmd/meroxa/builder/builder.go

```diff
@@ -650,7 +650,7 @@ func buildCommandWithSubCommands(cmd *cobra.Command, c Command) {
 	}
 }
 
-func hasFeatureFlag(flags []string, f string) bool {
+func HasFeatureFlag(flags []string, f string) bool {
```

Comment:
I don't think this function needs to be exported.

Reply:
forgot to undo this from a previous iteration of the code... fixed.